examples/version/ent/internal/schema.go

```diff
@@ -4,6 +4,7 @@
 
 // Code generated by entc, DO NOT EDIT.
 
+//go:build tools
```

Comment:
gofmt adds an equivalent `//go:build` constraint when encountering the older syntax (`// +build ...`).
The old syntax should be removed on Go 1.18.